src/viam/ur/module/ur_arm.cpp

```diff
@@ -796,6 +803,28 @@ void URArm::keep_alive_() {
 void URArm::move_(std::shared_lock<std::shared_mutex> config_rlock, std::list<Eigen::VectorXd> waypoints, const std::string& unix_time) {
     auto our_config_rlock = std::move(config_rlock);
 
+    if (reject_move_request_threshold_deg) {
+        auto joint_pos = [&]() {
+            auto joint_pos = get_joint_positions_(our_config_rlock);
+            auto vector = Eigen::VectorXd(joint_pos.size());
+            std::transform(std::make_move_iterator(joint_pos.begin()),
+                           std::make_move_iterator(joint_pos.end()),
+                           vector.begin(),
+                           degrees_to_radians<double>);
+            return vector;
+        }();
+
+        auto delta_pos = (waypoints.front() - joint_pos);
+
+        if (radians_to_degrees(delta_pos.lpNorm<Eigen::Infinity>()) > *reject_move_request_threshold_deg) {
+            std::stringstream err_string;
+            err_string << "rejecting move request : difference between starting trajectory position and joint position is above threshold "
+                       << radians_to_degrees(delta_pos.lpNorm<Eigen::Infinity>()) << " > " << *reject_move_request_threshold_deg;
```

Comment:
[opt] might be worth logging the waypoints and joint_pos too. that way users can do a lil bit of debugging without needing to go digging